receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -53,6 +55,95 @@ func (s *brokerScraper) shutdown(context.Context) error {
 	return nil
 }
 
+func (s *brokerScraper) scrapeConsumerFetch(now pcommon.Timestamp, allMetrics saramaMetrics, brokerID int64) {
+	key := fmt.Sprint("consumer-fetch-rate-for-broker-", brokerID)
```

Comment:
Lets move string prefixes to constants to avoid potential issues with typos.

Reply:
moved them into constant array, helpful for the code reduction as well